.travis.yml

```diff
@@ -1,23 +1,20 @@
 language: cpp
-cache: ccache
+cache:
+  ccache: true
+  directories:
+    - CachedCMakeCache # Avoid rediscovery of types sizes
+    - Testing          # Store ctest cost data
 
 # Use Linux by default
 os: linux
 dist: bionic
 
-# TODO: add ARM64, PPC, IBM builds when we find CMake for them
+# TODO: add IBM builds when we find CMake for them
+#       add PowerPC builds when sqlite3 crashing is fixed
 
 jobs:
   include:
     # OSX workers are slower to start up.  Having these first in the build matrix makes travis faster overall.
-    - name: "AppleClang - Release"
-      os: osx
-      osx_image: xcode11.3
-      before_install:
-        - HOMEBREW_NO_AUTO_UPDATE=1 brew install ccache
-      env: &Release
-        - TRAVIS_CUBERITE_BUILD_TYPE=Release
-
```

Comment:
It got "replaced" by arm, I thought since Travis is so generous giving us all this free compute time we should avoid running too many. Release Mac is a bit redundant when we have Debug Mac and Debease Linux.